tailwind.config.js

```diff
@@ -29,15 +30,22 @@ module.exports = {
         "4.5" : "1.125rem"             //18px
       },
       fontFamily: {
-        'Neufile': ['Neufile', 'sans-serif'],        
+        'Neufile': ['Neufile', 'sans-serif'],      
       },
       spacing:{
         "24.5" : "6.1513rem", //98.4208px
+        '0.75': '0.188rem', //3px
       },
       width:{
         "11.5/12" : "91%",      // width on desktop
         "11.8/12" : "93%",      //width on tablet
         "10.5/12" : "90%"      //width on mobile
+      },
+      height: {
+        "fit": "fit-content",
```

Comment:
What do you mean by  height: {
        "fit": "fit-content",
}
?

Reply:
That's a CSS property kindly google it